test/AlixBa/Addic7edSubtitles/Helpers/EpisodeTest.php

```diff
@@ -10,9 +10,9 @@ function testCapitalizedFileName()
 
         $this->assertEquals('The.Big.Bang.Theory', $episode->showName);
         $this->assertEquals('thebigbangtheory', $episode->sanitizedShowName);
-        $this->assertEquals('08', $episode->season);
-        $this->assertEquals('18', $episode->ep);
-
+        $this->assertEquals(8, $episode->season);
+        $this->assertEquals(18, $episode->ep);
+        //$this->assertEquals(['lol'], $episode->groups); // XXX shouldn't this contain only lol ?
         $this->assertEquals(['hdtv', 'x264'], $episode->tags);
```

Comment:
Plus, you can use `public function inGroups($group): Boolean` function, if you want to assert something.